src/templates/program-page.js

```diff
@@ -447,16 +447,12 @@ export const query = graphql`query ($id: String) {
             }
             relationships {
               field_media_image {
-                localFile {
-                  publicURL
```

Comment:
Assuming publicURL is not in use anywhere (I couldn't find a reference, so I'm thinking it's fine to remove it).